filters/parser.go

```diff
@@ -43,7 +43,7 @@ selectors := selector ("," selector)*
 selector  := fieldpath (operator value)
 fieldpath := field ('.' field)*
 field     := quoted | [A-Za-z] [A-Za-z0-9_]+
-operator  := "==" | "!=" | "~="
+operator  := "==" | "!=" | "~=" | "<" | ">"
```

Comment:
I intentionally don't support `<=` and `>=` since both of them can be expressed by using `<`  or `>`.